rslib/Makefile

```diff
@@ -1,4 +1,7 @@
-SHELL := bash
+SHELL := /bin/bash
+ECHOCMD := /bin/echo -e
+FIND_EXEC := $(if $(wildcard /bin/find),,/usr)/bin/find
```

Comment:
rather than "ECHOCMD" and "FIND_EXEC", perhaps just "ECHO" and "FIND".

And wouldn't the following be clearer?

```
$(if $(wildcard /bin/find),/bin/find,/usr/bin/find)
```

Reply:
But for me both were fine, but yes, this is clear. I am going to update it.